content/influxdb/v2.1/security/tokens/create-token.md

```diff
@@ -15,26 +15,90 @@ Create API tokens using the InfluxDB user interface (UI), the `influx`
 command line interface (CLI), or the InfluxDB API.
 
 {{% note %}}
-Tokens are visible to the user who created the token.
-{{% oss-only %}}Users who own a token with Operator permissions also have access to all tokens.{{% /oss-only %}}
+
+{{% oss-only %}}Tokens are visible to the user who created the token. Users who own a token with Operator permissions also have access to all tokens.
 Tokens stop working when the user who created the token is deleted.
+
 **We recommend creating a generic user to create and manage tokens for writing data.**
+{{% /oss-only %}}
```

Comment:
Does this format correctly? You may need to add a newline and a blank line after the opening shortcode tag. If it does format correctly, I'm fine with it.

```suggestion
{{% oss-only %}}

Tokens are visible to the user who created the token. Users who own a token with Operator permissions also have access to all tokens.
Tokens stop working when the user who created the token is deleted.

**We recommend creating a generic user to create and manage tokens for writing data.**

{{% /oss-only %}}
```

Reply:
It does format correctly. When I tried with a newline then it added an unwanted blank line.